src/main/java/org/walkmod/OptionsBuilder.java

```diff
@@ -30,26 +36,49 @@
  * @author abelsromero
  */
 public class OptionsBuilder {
+	private static final String DEFAULT_PATH = ".";
 
 	/**
-	 * Options instance
+	 * Collected options
 	 */
-	private Options options;
+	private final Map<String, Object> options = new HashMap<String, Object>();
```

Comment:
Shall I move the bulk of the OptionsBuilder into Options, change the builder-factory methods to the classic "builder()" and leave just the current public methods in OptionsBuilder delegating to the new ones and becoming deprecated?
If we think no-one outside the known walkmod universe uses them we can just remove them.